test/CircleTest.cc

```diff
@@ -0,0 +1,43 @@
+#include "CAD/Circle.hh"
+#include "CAD/Vec3d.hh"
+#include "opengl/StyledMultiShape2D.hh"
+#include <iostream>
+#include "opengl/GrailGUI.hh"
+#include "opengl/util/Camera.hh"
+#include <vector>
+
+using namespace std;
+
+class CircleTest : public GLWin {
+  private:
+    Style *baseGraphStyle;
+
+  public:
+    CircleTest() : GLWin(0x000000, 0xCCCCCC, "CircleTest") {}
+
+  void init(){
+    baseGraphStyle = new Style("TIMES", 12, 1, 0, 0, 0, 0, 0, 0);
+    baseGraphStyle->setLineWidth(10);
+  
+    //set the camera
+    MainCanvas *c = currentTab()->getMainCanvas();
+    Camera* cam = c->setLookAtProjection(0, 0, 30, 0, 0, 0, 0, 1, 0);
+    c->setProjection(cam->getViewProjection());
+
+    Vec3D a(1,1,1);
+    Vec3D b(2,3,4);
+
+    Circle* firstCircle = new Circle(1, a, b, c, baseGraphStyle);
```

Comment:
For some reason, these seem to miss drawing the last segment of the circle, so the circle almost connects but doesn't. I'll have to check out Circle.cc and see if there's a fix.

Reply:
This may be a need to switch to a different GL drawing method, or we may need to copy the first element to the last.